recipes/onmail/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "onmail",
+  "name": "onMail",
+  "version": "1.0.2",
```

Comment:
Please force commit this back to `1.0.0` since this is still in the PR

Reply:
I wanted to, but pnpm package is forcing me to have a version bump, Is there a way to avoid that?

Reply:
please fix this point alone - and we can approve the PR.

Reply:
`git commit --noverify`